sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -215,21 +215,14 @@ There are a number of files that affect the build configuration:
 Etiquette
 ~~~~~~~~~
 
-CI services provide finite free build workers to Open Source projects. A few
-considerations will help you be considerate of others needing these limited
+CI services provide finite free build workers to Open Source projects. The below
+consideration will help you be considerate of others needing these limited
 resources:
 
 * Group commits into meaningful chunks of work before pushing to GitHub (i.e.
```

Comment:
Removing the second item makes this a one-item bullet list. I would recommend rewriting this into one cohesive paragraph and not have a bullet list at all.